tests/NuGet.Services.Validation.Orchestrator.Tests/ValidatorStateServiceFacts.cs

```diff
@@ -28,6 +28,9 @@ public class ValidatorStateServiceFacts
         private static readonly Guid ValidationId = new Guid("fb9c0bac-3d4d-4cc7-ac2d-b3940e15b94d");
         private static readonly Guid OtherValidationId = new Guid("6593BD33-ABC0-4049-BDCF-915807F1D2B3");
         private const string NupkgUrl = "https://example/nuget.versioning/4.3.0/package.nupkg";
+        private static readonly Uri inputUrl = new Uri("https://example/vsix.versioning/4.3.0/package.vsix");
```

Comment:
Please capitalize statics:

```suggestion
        private static readonly Uri InputUrl = new Uri("https://example/vsix.versioning/4.3.0/package.vsix");
```